### DISCUSSION TITLE
Album description field formatting

### DISCUSSION THREAD

**Discussion by haivala** [LycheeOrg/Lychee#2334] [Ideas] [answered]
### Enhancement

It would be nice if it would accept more than just text. 
html or markdown or anything with links and new lines

**Comment by ildyria** [comment, marked as answer]
On its way: https://github.com/LycheeOrg/Lychee/pull/2335

Fun fact, integration was already here, just missing the styling.

**Comment by haivala** [comment]
How about adding this to individual image description too?

**Reply by ildyria** [reply]
Did you try ? 😅

**Reply by haivala** [reply]
yes. This works in album but not in pic desc `[Music](https://chiph.art/musiikki/chiphead/e-tyyli-19)  `

**Reply by ildyria** [reply]
Weird. I will have a deeper look because it was working on my side for bold, italic etc.

**Reply by ildyria** [reply]
On master:
![image](https://github.com/LycheeOrg/Lychee/assets/627094/be3e0ae9-b9de-4cad-a1d5-811f065b52e7)

**Reply by haivala** [reply]
ok. I'll try again!

**Reply by haivala** [reply]
Hmm. Does it matter if the pic is actually mp4 video?

**Reply by haivala** [reply]
Actually the formatting works like you showed in the sidebar but it does not work if it is on the pic under the title at the left

**Reply by haivala** [reply]
@ildyria did you check that?

**Reply by ildyria** [reply]
I will try to have a look. :+1:

**Reply by ildyria** [reply]
https://github.com/LycheeOrg/Lychee/pull/2347
